app/src/main/java/com/hadiyarajesh/notex/database/entity/Reminder.kt

```diff
@@ -10,7 +10,7 @@ import java.time.Instant
 @JsonClass(generateAdapter = true)
 data class Reminder(
     @PrimaryKey(autoGenerate = true)
-    val reminderId: Long? = null,
+    val reminderId: Long? =null,
```

Comment:
why this change? the previous formatting is good.

check your format setting in your IDE.